blocks/components/editable/index.js

```diff
@@ -103,10 +110,25 @@ export default class Editable extends wp.element.Component {
 
 	updateContent() {
 		const bookmark = this.editor.selection.getBookmark( 2, true );
-		this.editor.setContent( this.props.value );
+		this.setContent( this.props.value );
 		this.editor.selection.moveToBookmark( bookmark );
 	}
 
+	setContent( content ) {
+		if ( ! content ) {
+			return '';
+		}
+
+		content = wp.element.renderToString( content );
```

Comment:
Not the intent of the change, but a trade-off that could help performance slightly is change to retrieve content from editor instance as raw formatted in 075237ef5d3947a9f784c27580bf6ba9a9193a4e.